Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -711,7 +711,13 @@ open class NavigationMapView: UIView {
     
     // MARK: Building Extrusion Highlighting
     
-    @objc dynamic public var buildingDefaultColor: UIColor = .defaultBuildingColor
+    @available(*, deprecated, message: "NavigationMapView.buildingDefaultColor is no longer used.")
+    @objc dynamic public var buildingDefaultColor: UIColor = .clear
```

Comment:
Looks live we never used this value. Will have to deprecate.

Reply:
It looks like #2535 introduced this color to style an all-buildings layer. Do we still support that?

https://github.com/mapbox/mapbox-navigation-ios/blob/fb5e92d51961ddf08acf46c5169b553f6d52dcb5/MapboxNavigation/NavigationMapView.swift#L1416-L1419

Reply:
Good catch. I think that we've missed it during porting to Maps SDK v10. I think that it should be used when all buildings should be highlighted (if building are found at coordinates passed by user, they will be highlighted in `buildingHighlightColor`, those, which were not found will be highlighted in `buildingDefaultColor`).